presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoDriverUri.java

```diff
@@ -149,6 +152,24 @@ public Properties getProperties()
         return properties;
     }
 
+    public Optional<String> getTimeZoneID()
+            throws SQLException
+    {
+        Optional<String> timezone = TIMEZONE_ID.getValue(properties);
+
+        if (timezone.isPresent()) {
+            try {
+                // Validate timezone and return it.
+                ZoneId.of(timezone.get());
+                return timezone;
+            }
+            catch (DateTimeException e) {
+                throw new SQLException("Specified TimeZoneID is not supported: " + timezone.get(), e);
+            }
+        }
+        return timezone;
```

Comment:
We can modify the return type of this method to String and return `TimeZone.getDefault().getID()` when the parameter is not set.